spyder_unittest/widgets/unittestgui.py

```diff
@@ -60,11 +60,19 @@ def is_unittesting_installed():
 
 
 class UnitTestWidget(QWidget):
-    """Unit testing widget."""
+    """
+    Unit testing widget.
+
+    Signals
+    -------
+    sig_processed_tests: Emitted when plugin finishes processing tests.
+    """
 
     DATAPATH = get_conf_path('unittest.results')
     VERSION = '0.0.1'
 
+    sig_processed_tests = Signal()
```

Comment:
@jitseniesen this could be something simpler like, `sig_finished`, just a suggestions ;-)